### PR TITLE
Add supports for Symfony 4

### PR DESCRIPTION
**Pull request by mcg-web**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,27 +19,35 @@ matrix:
         - php: 5.5
           env: SYMFONY_VERSION=2.8.*
         - php: 5.6
-          env: SYMFONY_VERSION=3.0.*
-        - php: 7.0
           env: SYMFONY_VERSION=3.1.*
```
Comment (renatomefi) [inline .travis.yml]: Maybe keep one 7.0 build?

File: .travis.yml [hunk 2]
```diff
@@ -15,27 +19,35 @@ matrix:
...
         - php: 7.1
-          env: SYMFONY_VERSION=3.2.*
-        - php: hhvm
+          env: PHP_CS_FIXER=true
+        - php: 7.1
```
Comment (renatomefi) [inline .travis.yml]: How about adding prefer-lowest here as well?

Comment (mcg-web) [reply .travis.yml]: prefer-lowest fails because of phpunit 4, I'll added in a next PR

Comment (renatomefi) [reply .travis.yml]: If the problem is only the TestCase class you can set the minimum phpunit to version 4.8.35
https://github.com/sebastianbergmann/phpunit/blob/master/ChangeLog-4.8.md

**Pull request merged**